metrics/cli/test/sentry/adapter/sentry_cli_service_adapter_test.dart

```diff
@@ -0,0 +1,415 @@
+// Use of this source code is governed by the Apache License, Version 2.0
+// that can be found in the LICENSE file.
+
+import 'package:cli/sentry/adapter/sentry_cli_service_adapter.dart';
+import 'package:cli/sentry/cli/sentry_cli.dart';
+import 'package:cli/sentry/model/sentry_project.dart';
+import 'package:cli/sentry/model/sentry_release.dart';
+import 'package:cli/sentry/model/source_map.dart';
+import 'package:cli/sentry/strings/sentry_strings.dart';
+import 'package:mockito/mockito.dart';
+import 'package:test/test.dart';
+
+import '../../test_utils/matchers.dart';
+import '../../test_utils/prompter_mock.dart';
+
+// ignore_for_file: avoid_redundant_argument_values
+
+void main() {
+  group('SentryCliServiceAdapter', () {
+    const path = 'path';
+    const extensions = ['js'];
+    const releaseName = 'releaseName';
+    const organizationSlug = 'organizationSlug';
+    const projectSlug = 'projectSlug';
+    const enterOrganizationSlug = SentryStrings.enterOrganizationSlug;
+    const enterReleaseName = SentryStrings.enterReleaseName;
+    const dsn = 'dsn';
+
+    final sentryCli = _SentryCliMock();
+    final prompter = PrompterMock();
+    final sentryService = SentryCliServiceAdapter(
+      sentryCli: sentryCli,
+      prompter: prompter,
+    );
+    final sentryProject = SentryProject(
+      organizationSlug: organizationSlug,
+      projectSlug: projectSlug,
+    );
+    final sentryRelease = SentryRelease(
+      name: releaseName,
+      project: sentryProject,
+    );
+    final sourceMap = SourceMap(path: path, extensions: extensions);
+    final stateError = StateError('test');
+
+    tearDown(() {
```

Comment:
I believe it would be great to place the `tearDown` right above the tests. What do you think?